target/linux/mediatek/filogic/base-files/etc/board.d/01_leds

```diff
@@ -29,6 +29,12 @@ xiaomi,redmi-router-ax6000-stock|\
 xiaomi,redmi-router-ax6000-ubootmod)
 	ucidef_set_led_netdev "wan" "wan" "rgb:network" "wan"
 	;;
+zbtlink,zbt-z8103ax)
+	ucidef_set_led_netdev "lan1" "lan1" "green:status" "lan1" "link tx rx"
+	ucidef_set_led_netdev "lan2" "lan2" "green:status" "lan2" "link tx rx"
+	ucidef_set_led_netdev "lan3" "lan3" "green:status" "lan3" "link tx rx"
```

Comment:
Why LAN 1-3 is present here but LAN4 not?

Reply:
> The router has the following physical ports:
> 
> * Lan1
> * Lan2
> * Lan3
> * Wan

Ok. My suggestion is to drop lan4 from dts in this case,

> I based my DTS file here: https://forum.openwrt.org/t/zbt-z8103ax-d-upgrading-from-21-02-to-23-05/174582/7

Oh no, this is a dts from ROOTer. lol I thought that ROOTer is made for devices with usb port only.